bin/post_compile

```diff
@@ -80,21 +97,23 @@ heroku-install-cabot-plugins() {
   done
 }
 
+heroku-put-scripts-on-path() {
+  # A better way of doing this perhaps?
+  ln -sf /app/.heroku/vendor/node-4.2.6/bin/* /app/.heroku/python/bin/
```

Comment:
Worth checking what the node directory is before symlinking. Is there a reason why this didn't work as is? I set the path below on line `107`.